src/Tests/Microsoft.Diagnostics.Monitoring.OpenApiGen.UnitTests/Microsoft.Diagnostics.Monitoring.OpenApiGen.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Do all tfm's produce the same openapi output?

Reply:
No, in fact, that was broken recently and fix. The generator and the tests build for all TFMs but only the `net6.0` TFM is actually run in tests.